Packs/Automox/Integrations/Automox/Automox.py

```diff
@@ -0,0 +1,612 @@
+    limit = args.get('limit', None)
+    page = args.get('page', None)
+
+    result = client.list_vulnerability_sync_batches(org_id, limit, page)
+
+    return CommandResults(
+        outputs_prefix="Automox.VulnSyncBatches",
+        outputs_key_field='id',
+        outputs=result['data'],
+    )
+
+def list_vulnerability_sync_tasks(client: Client, args: Dict[str, Any]) -> CommandResults:
+    org_id = args.get('org_id', None) or DEFAULT_ORG_ID
+    batch_id = args.get('batch_id', None)
+    status = args.get('status', None)
+    limit = args.get('limit', None)
+    page = args.get('page', None)
+
+    result = client.list_vulnerability_sync_tasks(org_id, batch_id, status, limit, page)
+
+    excluded_keys = [
+        'partner_user_id',
+    ]
+
+    try:
+        for task in result['data']:
+            for key in excluded_keys:
+                task.pop(key)
+    except Exception as e:
+        demisto.error(f"Key '{key}' not found in Automox Vulnerability Sync task list response.")
+
+    return CommandResults(
+        outputs_prefix="Automox.VulnSyncTasks",
+        outputs_key_field='id',
+        outputs=result['data'],
+    )
+
+def run_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    org_id = args.get('org_id', None) or DEFAULT_ORG_ID
+    device_id = args.get('device_id', None)
+    command_type_name = args.get('command', None)
+    args = args.get('args', None)
+
+    payload = {
+        'command_type_name' : command_type_name,
+        'args' : args
+    }
+
+    client.run_command(org_id, device_id, payload)
+
+    return CommandResults(
+        mark_as_note=True,
+        readable_output=f"Command: {command_type_name} successfully sent to Automox device ID: {device_id}"
+    )
+
+def update_device(client: Client, args: Dict[str, Any]) -> CommandResults:
+    org_id = args.get('org_id', None) or DEFAULT_ORG_ID
+    device_id = args.get('device_id', None)
+    server_group_id = args.get('server_group_id', None)
+    custom_name = args.get('custom_name', None)
+    tags = args.get('tags', None)
+    ip_addrs = args.get('ip_addrs', None)
+    exception = args.get('exception', None)
+
+    payload = {
+        "server_group_id" : server_group_id,
+        "ip_addrs" : ip_addrs,
+        "exception" : exception,
+        "tags" : tags,
+        "custom_name" : custom_name,
+    }
+
+    result = client.update_device(org_id, device_id, payload)
+
+    return CommandResults(
+        outputs_prefix="Automox",
+        outputs_key_field='',
+        outputs=result,
+    )
+
+def update_group(client: Client, args: Dict[str, Any]) -> CommandResults:
+    #server_group_id, custom_name, tags, ip_addrs, exception=False
+    org_id = args.get('org_id', None) or DEFAULT_ORG_ID
+    group_id = args.get('group_id', None)
+    color = args.get('color', None)
+    name = args.get('name', None)
+    notes = args.get('notes', None)
+    parent_server_group_id = args.get('parent_server_group_id', None)
```

Comment:
I believe this might fail. I thought we couldn't set a parent server group to none 🤔 . Worth testing to make sure

Reply:
Good catch

Reply:
In the past what I've done on update is check what current ID is and set it (I think). Then when we implement `create_group` if not defined I pull the default group ID for the org and set it for the user.